libs/api/domains/rsk-company-info/src/lib/graphql/models/rskCompanySearchItems.model.ts

```diff
@@ -0,0 +1,24 @@
+import { Field, ObjectType, ID } from '@nestjs/graphql'
+import { RskCompanyLink } from './rskCompanyLink.model'
+import { RskCompanySearchItem } from './rskCompanySearchItem.model'
+
+@ObjectType()
+export class RskCompanySearchItems {
```

Comment:
Should follow a simplified version of Relays connection pattern:
https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#fetching-array-of-items